tapioca.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency("pry-byebug")
   spec.add_development_dependency("rspec", "~> 3.7")
   spec.add_development_dependency("rubocop", "~> 0.70.0")
-  spec.add_development_dependency("sorbet")
+  spec.add_development_dependency("sorbet", "~> 0.4.4813")
```

Comment:
This is only a development dependency, while sorbet-static was a runtime dependency. They behave in different ways. I think what we want is to lock the runtime dependency version. That way users can't install this gem with a version below the one we require.

Also, theoretically you don't need to change the gemspec on this commit, unless this gem only works now with `sorbet >= 0.4.4813`. Since I see no changes in the code on this PR, I think that is not the case.